public/app/features/serviceaccounts/ServiceAccountPage.tsx

```diff
@@ -67,8 +66,9 @@ export const ServiceAccountPageUnconnected = ({
   const [isTokenModalOpen, setIsTokenModalOpen] = useState(false);
   const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
   const [isDisableModalOpen, setIsDisableModalOpen] = useState(false);
+  const { id = '0' } = useParams();
```

Comment:
why do we default to `'0'` here? 🤔

Reply:
Probably forgot to remove after testing. But tbh it doesn't really matter what the default is, if the param is not there, this component is not going to be rendered.